CONTRIBUTING.md

```diff
@@ -130,42 +151,49 @@ Every Python and Cython file should only include the following license header:
 # See the License for the specific language governing permissions and
 # limitations under the License.
 ```
+
 The copyright year should be updated every calendar year.
 
 ## Security
 
 ### Bandit
 
-We use [Bandit](https://github.com/PyCQA/bandit) to find common security issues
-in Python code.
+We use [Bandit*](https://github.com/PyCQA/bandit) to find common security issues
+in the Python* code.
 
-Install: `pip install bandit`
+To install, run:
+```bash
+pip install bandit
+```
 
-- Revision: `1.7.0`
+Revision: `1.7.0`
 
-Run before each commit: `bandit -r dpctl -lll`
+Before each commit, run:
+```bash
+bandit -r dpctl -lll
+```
 
 ## Code Coverage
 
-Code coverage for both C and Python sources in dpctl is generated for each
+Code coverage, for both C and Python sources in dpctl, is generated for each
 pull request (PR). A PR cannot be merged if it leads to a drop in the code
-coverage by more than five percentage points. *Ergo, do not forget to write
-unit tests for your changes.* To check the code coverage for your code, follow
-these steps:
+coverage by more than five percentage points. Therefore, write
+unit tests for your changes. 
+
+To check the code coverage for your code, follow these steps:
 
 1. Install dependencies for C/C++ source.
 
-    For C/C++ source we require `lcov`, `llvm-cov`, and `llvm-profdata`. Note
-    that `llvm-cov` and `llvm-profdata` should be version 11.0 or higher. If you
-    have multiple `llvm-cov` tools installed, most likely because you have
-    multiple llvm installations, you should set the `LLVM_TOOLS_HOME`
+    For C/C++ source, we require `lcov`, `llvm-cov`, and `llvm-profdata`. `llvm-cov` and `llvm-profdata` 
+    should be version 11.0 or higher. If you
```

Comment:
```suggestion
    For C/C++ source, we require `lcov`, `llvm-cov`, and `llvm-profdata`. `llvm-cov` and `llvm-profdata` 
    that are version 11.0 or higher. If you
```

Reply:
> How about just:

Must better! Thanks @diptorupd

Reply:
Is it okay if I change it to: 
```suggestion
    For C/C++ source,  `lcov`, `llvm-cov` (>=11.0), and `llvm-profdata` (>=11.0) are required.
```

Reply:
👍

Reply:
Please make sure that the trailing "If you" does not get lost.